go-controller/pkg/libovsdbops/portgroup.go

```diff
@@ -161,12 +161,26 @@ func DeleteACLsFromPortGroupOps(nbClient libovsdbclient.Client, ops []libovsdb.O
 		return ops, nil
 	}
 
+	pACL := func(item *nbdb.ACL) bool {
+		for _, acl := range acls {
+			if isEquivalentACL(item, acl) {
+				return true
+			}
+		}
+		return false
+	}
+
+	matchAcls, err := FindACLsWithPredicate(nbClient, pACL)
```

Comment:
I would rather have in `acl.go`:
```
func GetACLs(nbClient libovsdbclient.Client, acls []*nbdb.ACL) ([]*nbdb.ACL, error) {
    return FindACLsWithPredicate(nbClient, func(item *nbdb.ACL) bool {
        for _, acl := range acls {
             if isEquivalentACL(item, acl) { 
                 return true 
             }
         }
         return false
    })
}
```

and have the caller of `DeleteACLsFromPortGroupOps` call `GetACLs` itself before `DeleteACLsFromPortGroupOps`.

Reply:
What about having 2 functions: one that takes a list of acl UUIDs, and the other one that does an additional search? There is only 1 function call that needs the latter

Reply:
Actually, if you look for a usage, it's just to clear ACLs from the port group that is about to get deleted https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/ovn/policy.go#L512-L520
But port group can be deleted without additional ACL cleanup (and ACLs themselves are not deleted, not sure why), but to fix the problem we can just remove that `DeleteACLsFromPortGroupOps` call (and maybe still change `DeleteACLsFromPortGroupOps` args to `aclUUIDs` to avoid confusion in the future)